src/services/permissions/resources.ts

```diff
@@ -0,0 +1,14 @@
+export const resouces = {
+  topology: "topology",
+  health: "health",
+  incidents: "incidents",
+  config: "config",
+  logs: "logs",
+  "settings.users": "settings.users",
+  "settings.teams": "settings.teams",
+  "settings.rules": "settings.rules",
+  "settings.config_scraper": "settings.config_scraper",
```

Comment:
The settings are linked to the primary property e.g. if config.disable == true hide both configs and config scraper tabs